on-prem-docs/README.MD

```diff
@@ -0,0 +1,75 @@
+# Chef Habitat Builder on-prem  - Website
+
+A new static website for Chef Habitat on-prem is based on [Hugo](https://gohugo.io), the Go-powered static site generator.
+
+## Getting Started
+
+### One Time Install
+
+1. Disable smart quotes. OS X tries to use smartquotes by default when editing Hugo docs. You want smartquotes disabled regardless if you are writing code. Here's how [you can disable it](http://www.iclarified.com/38772/how-to-disable-curly-quotes-in-mac-os-x-mavericks).
```

Comment:
Might want to mention these steps are for OS X only, and that other platforms will have different steps to install Hugo

Reply:
`hab pkg install core/hugo` 😉

Reply:
I've added common installation methods and linked to the Hugo docs. I'm leaving out the `hab pkg install` method because I don't want to have people install hab as a pre-requisite for installing hab. Although, yes, it can also be done this way. ;-)